tests/conftest.py

```diff
@@ -1,13 +1,22 @@
 import csv
+import logging
+import os
 import sqlite3
 from pathlib import Path
 
+import psycopg2
 import pytest
 
+assert "POSTGRES_DSN" in os.environ, "Postgres dsn not set"
```

Comment:
Isn't this going to fail when `POSTGRES_DSN` is not set?

Reply:
> Isn't this going to cause the entire test suite to fail when `POSTGRES_DSN` is not set?


Yes that's the whole point of the pr, be more flexible, not everyone has postgresql running locally, currently the test suite fails if you're not running a local postgresql that listens on 5432. 

> Running `pytest` should work, and should not require any specific environment variables to be present.

It does work, the pr could default the env var to localhost 5432 if not set though.